index.html

```diff
@@ -0,0 +1,361 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 1</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="body">
+        <div class="nav">
+            <ul class="navbar-nav">
+                <li class="nav-item">
```

Comment:
layout is shifting while hover, add a transparent border, or border with nav color